src/client/pkg/config/config.go

```diff
@@ -221,3 +221,19 @@ func (c *Config) Write() error {
 	value = proto.Clone(c).(*Config)
 	return nil
 }
+
+func WritePachTokenToConfig(token string) error {
```

Comment:
Refactored this out of the auth command-line in the config client package, so the auth test utils can configure an auth token for pachctl. This is useful for debugging when tests fail, and prevents clusters from getting stuck in an unusable state.